rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -904,6 +909,159 @@ void test_VerifyTXFeeIsSpentEquallyForBatchedPegouts_three_pegouts() {
         Assertions.assertEquals(inputsValue.minus(totalPegoutAmount), changeOutput.getValue());
     }
 
+    @ParameterizedTest()
+    @MethodSource("providePegoutMinimumParameters")
+    void buildBatchedPegouts_whenSpendableUXTO_shouldBuildTransactionSuccessfully(Coin feePerKb, Coin minPegoutValue) {
+        // build federation with 9 members
+        Integer[] memberPKs = new Integer[]{ 100, 200, 300, 400, 500, 600, 700, 800, 900 };
+        Federation fed = FederationTestUtils.getFederation(memberPKs);
+        // list of peg-out requests with the given minimum peg-out value
+        List<ReleaseRequestQueue.Entry> entries = Arrays.asList(
+            createTestEntry(1000, minPegoutValue));
+        // list of utxos that contains one utxo with a value of 1 BTC
+        List<UTXO> utxos = Arrays.asList(
+            new UTXO(mockUTXOHash("utxo"), 0, Coin.COIN, 0, false, fed.getP2SHScript()));
+        // the federation wallet, without flyover support
+        Wallet fedWallet = BridgeUtils.getFederationSpendWallet(
+            Context.getOrCreate(networkParameters),
+            fed,
+            utxos,
+            false,
+            mock(BridgeStorageProvider.class)
+        );
+        // build release transaction builder for current fed
+        when(activations.isActive(ConsensusRule.RSKIP201)).thenReturn(true);
```

Comment:
Just in case, let's use ActivationConfigForTest class. And get the `all` object, that includes all RSKIPs active

Reply:
How would it look in code? If I understand correctly, I should still mock `isActive(ConsensusRule.RSKIP201)` right?